### ISSUE TITLE
Does DVC work with SFTP as well?

### ISSUE THREAD

**Issue by haimat** [iterative/dvc-ssh#39]
I want to provide full read+write access to our DVC remote, but without giving away the password of that SSH user to our DVC users. I don't want to solve that with a Linux group and multiple users within that group, since that group just has read-only access to the DVC data folder. Can I thus make the main user of the DVC data folder on the server an SFTP-only user, without a valid login shell? If not, what other alternatives do I have?

**Comment by efiop** [comment]
Yes, it should work. Please give it a try and let us know how it goes.
